setup.py

```diff
@@ -85,14 +85,15 @@
 install_requires = [
     'Flask-BabelEx>=0.9.3',
     'Flask-Breadcrumbs>=0.3.0',
+    'Flask-Login>=0.4.0',
```

Comment:
Deterministic request? What do you mean?
Anyway, could we open a ticket for it? Thanks.
I'm quite busy now with the CDS sprint and this PR with this [PR for deposit](https://github.com/inveniosoftware/invenio-deposit/pull/143) are necessary to us to follow working.
When I'll be free I'll better check.